services/galley/test/integration/API/MLS/Util.hs

```diff
@@ -257,7 +258,9 @@ mlscli :: HasCallStack => ClientIdentity -> [String] -> Maybe ByteString -> MLST
 mlscli qcid args mbstdin = do
   bd <- State.gets mlsBaseDir
   let cdir = bd </> cid2Str qcid
-  liftIO $ spawn (proc "mls-test-cli" (["--store", cdir </> "store"] <> args)) mbstdin
+  liftIO $ do
+    -- Imports.setEnv "RUST_BACKTRACE" "full"
+    spawn (proc "mls-test-cli" (["--store", cdir </> "store"] <> args)) mbstdin
```

Comment:
Dead code?

Reply:
Thanks for spotting this! Removed in commit 9e1087534.